galaxy_ng/app/management/commands/assign-permission.py

```diff
@@ -4,10 +4,10 @@
 from django.contrib.auth.models import Permission
 from django.core.management import BaseCommand, CommandError
 from django.core.exceptions import ObjectDoesNotExist
+from django.utils.translation import gettext_lazy as _
 
 User = get_user_model()
 
-
 class Command(BaseCommand):
     """
     Django management command for assigning permissions to groups.
```

Comment:
if the goal of translating the api is ensure translated strings get passed to the UI, then we probably don't need to translate the management commands `galaxy_ng/app/management/commands/*` since they are not part of the api